public/data/resources.json

```diff
@@ -9,7 +9,7 @@
   "locationCountry": "France",
   "image": "images/social-share3.jpg",
   "keywords": "sunnytech, évènement, event, montpellier, Sunny, Tech, programming, android, chrome, developers, web, cloud, conférence, seatechandsun, docker, devops, ux, design, back, scala, fonctionnel",
-  "dates": "29 & 30 juin 2023",
+  "dates": "5 & 6 juillet 2023",
```

Comment:
2024 ?

Reply:
Yes, my bad. By the way is the date correct ? it 5 & 6 or 4 & 5 ?

Reply:
4 & 5 effectivrmernt ! jeud et vendredi, le 6 c'est un samedi 📦